tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rh-sign-image
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Task to create internalrequests to sign snapshot components
+  params:
+    - name: snapshotPath
+      description: Path to the JSON string of the mapped Snapshot spec in the data workspace
+      type: string
+      default: "mapped_snapshot.json"
+    - name: dataPath
+      description: Path to the JSON string of the merged data to use in the data workspace
+      type: string
+      default: "data.json"
+    - name: requester
+      type: string
+      description: Name of the user that requested the signing, for auditing purposes
+    - name: requestTimeout
+      type: string
+      default: "180"
+      description: InternalRequest timeout
+  workspaces:
+    - name: data
+      description: workspace to read and save files
+  steps:
+    - name: sign-image
+      image: quay.io/hacbs-release/release-utils:a9603c0ae1982be5e1142d44765b1475adc7ad40
+      script: |
+        #!/usr/bin/env sh
+        #
+        SNAPSHOT_PATH=$(workspaces.data.path)/$(params.snapshotPath)
+        RUNDIR="$(workspaces.data.path)/$(context.taskRun.uid)"
+
+        if [ ! -d $RUNDIR ]; then
```

Comment:
Nit: With `mkdir -p`, you don't necessarily need the `if` - it will only create the dir if it doesn't exist. But if you do want the `if`, then the `-p` in `mkdir` is not needed.

Reply:
changed to `mkdir -p` without the conditional.